features/promotion/receiving_discount/applying_promotion_with_expiration_date.feature

```diff
@@ -10,28 +10,28 @@ Feature: Applying promotion with an expiration date
         And there is a promotion "Christmas sale"
         And this promotion gives "$10.00" discount to every order
 
-    @ui
+    @ui @api
     Scenario: Receiving a discount from a promotion which does not expire
         Given this promotion expires tomorrow
         When I add product "PHP T-Shirt" to the cart
         Then my cart total should be "$90.00"
         And my discount should be "-$10.00"
 
-    @ui
+    @ui @api
     Scenario: Receiving no discount from a valid but expired promotion
         Given this promotion has already expired
         When I add product "PHP T-Shirt" to the cart
         And my cart total should be "$100.00"
         And there should be no discount
 
-    @ui
+    @ui @api
     Scenario: Receiving a discount from a promotion which has already started
         Given this promotion has started yesterday
         When I add product "PHP T-Shirt" to the cart
         Then my cart total should be "$90.00"
         And my discount should be "-$10.00"
 
-    @ui
+    @ui @apig
```

Comment:
Hello,
What is `@apig` ?

```suggestion
    @ui @api
```

Reply:
🤭  @oallain its when i try to write git but im still in phpstorm window 😄

Reply:
:joy: it happens.